products/appengine/api.yaml

```diff
@@ -20,6 +20,10 @@ versions:
     base_url: https://appengine.googleapis.com/v1/
 scopes:
   - https://www.googleapis.com/auth/cloud-platform
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: App Engine Admin API
+    url: https://console.cloud.google.com/apis/library/appengine.googleapis.com/
 objects:
   - !ruby/object:Api::Resource
     name: 'FirewallRule'
```

Comment:
In the resource for `StandardApplicationVersion` we will need the field
`collection_url_key: 'versions'`
That will tell the InSpec plural resource how to unwrap the list of versions that it receives.

Reply:
I added this.

Reply:
Added